packages/@aws-cdk/aws-lambda/README.md

```diff
@@ -272,6 +272,32 @@ const fn = new lambda.Function(this, 'MyFunction', {
 See [the AWS documentation](https://docs.aws.amazon.com/lambda/latest/dg/concurrent-executions.html)
 managing concurrency.
 
+### Configuring AutoScaling
+
+You can configure autoscaling of provisioned concurrency on a function alias using the `autoScaleProvisionedConcurrency()` method:
+
+```ts
+const alias = new lambda.Alias(stack, 'Alias', {
+  aliasName: 'prod',
+  version,
+});
+
+// Configure autoscaling to track utilization
+alias.autoScaleProvisionedConcurrency({ minCapacity: 1, maxCapacity: 50 }).scaleOnUtilization({
```

Comment:
We try to use `addXxx` as the main verb. The mental model is "we are adding this to the definition of the alias". Also, keep names short and concise. I don't think there's a need to explicitly call out that this is for the provisioned concurrency given that's the only autoscaling we support.

```suggestion
alias.addAutoScaling({ minCapacity: 1, maxCapacity: 50 }).scaleOnUtilization({
```